spec/factories/groups.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+FactoryGirl.define do
+  factory :group do
+    size 1
+    status 'open'
```

Comment:
Shouldn't this actually be closed by default because a Group of size 1 is just the leader?